go/pkg/casng/pubsub.go

```diff
@@ -8,13 +8,9 @@ import (
 	"github.com/pborman/uuid"
 )
 
-// tag identifies a pubsub channel for routing purposes.
-// Producers tag messages and consumers subscribe to tags.
-type tag string
-
 // pubsub provides a simple pubsub implementation to route messages and wait for them.
 type pubsub struct {
-	subs    map[tag]chan any
+	subs    map[string]chan any
```

Comment:
What was the motivation to move from the `tag` alias to just `string`. Wouldnt the alias help ensure that the tags are valid?

Reply:
In a subsequent PR, I'll have code like `strings.Join(r.tags, "|")` in a log message. The custom types forces me to reallocate the array with an explicit string type. That or the use of generics sounds too much for a simple log message. The value of the custom type is entirely for readability as I don't expect it to have any special behaviour attached to it. It's treated as a string everywhere in this package.